examples/rust/udp-echo.rs

```diff
@@ -43,6 +44,9 @@ pub const AF_INET: i32 = windows::Win32::Networking::WinSock::AF_INET.0 as i32;
 #[cfg(target_os = "windows")]
 pub const SOCK_DGRAM: i32 = windows::Win32::Networking::WinSock::SOCK_DGRAM as i32;
 
+#[cfg(target_os = "windows")]
+use windows::Win32::Networking::WinSock::SOCKADDR;
+
```

Comment:
I think we've been trying to put all `use` statements at the top of the file, before anything else (like constant declarations).  And put them in the right place relative to the other `use` statements alphabetically.  I don't have a real strong opinion about this though.  @ppenna ?

Reply:
Use statements are like imports, so far, I have listed them in the top of the follow under the "Imports" section. I'm fine with whichever agreement that we come up with.

Reply:
Done, I moved the `use` statements to the top (sorted).